Annotations/AddAnnotation/MainWindow.xaml.cs

```diff
@@ -17,7 +18,11 @@ public MainWindow()
         {
             InitializeComponent();
             //Loads the document in PDF Viewer
-            pdfViewer.Load("../../F Sharp Succinctly.pdf");
+#if NETFRAMEWORK
+            pdfViewer.Load("../../Data/F Sharp Succinctly.pdf");
+#else
+            pdfViewer.Load("../../../Data/F Sharp Succinctly.pdf");
```

Comment:
Where is the data file located in this branch?

Reply:
Data file add in branch completed